src/activities/canal_lock/CanalLock.qml

```diff
@@ -46,6 +46,32 @@ ActivityBase {
             activity.start.connect(start)
             activity.stop.connect(stop)
         }
+        
+        IntroMessage {
+            id: message
+            anchors {
+                top: parent.top
+                topMargin: 10
+                right: parent.right
+                rightMargin: 5
+                left: parent.left
+                leftMargin: 5
+            }
+            z: 100
+            onIntroDone: {
+                anim.running = true
+                info.visible = true
+            }
+            intro: [
+                qsTr("Your goal is to get Tux across the canal lock to his get wooden logs, "
```

Comment:
to get his

Reply:
@AkshayCHD @divyam3897 Let it be `to get` not `to get his`/ `to get her`. Do normalize like `to get`